src/main/java/com/github/vkorobkov/jfixtures/loader/FixturesLoader.java

```diff
@@ -38,28 +45,45 @@ private boolean isFile(Path file) {
         return !Files.isDirectory(file);
     }
 
-    private boolean isYml(Path file) {
-        return file.toFile().getName().endsWith(".yml");
+    private boolean isYml(Path path) {
+        return Stream
+                .of(YAML_EXT, YML_EXT)
+                .anyMatch(ext -> getFileName(path).endsWith(ext));
     }
 
-    private boolean isNotConfig(Path file) {
-        return !file.toFile().getName().startsWith(".");
+    private boolean isNotConfig(Path path) {
+        return !getFileName(path).startsWith(".");
     }
 
     private Fixture loadFixture(Path file) {
-        String name = getFixtureName(file);
+        val name = getFixtureName(file);
         val baseColumns = config.columns().forTable(name);
         return new Fixture(name, new YmlRowsLoader(file, baseColumns));
     }
 
     private String getFixtureName(Path file) {
-        String separator = file.getFileSystem().getSeparator();
-        String relativePath = Paths.get(path).relativize(file).toString();
-        relativePath = relativePath.substring(0, relativePath.lastIndexOf(".yml"));
+        val separator = file.getFileSystem().getSeparator();
```

Comment:
Let's keep using types instead of `val` when types are short and don't have generics?